tiger/src/lib.rs

```diff
@@ -0,0 +1,1255 @@
+//! The [Tiger][1] hash function, implemented through [Digest][2]
+//!
+//! [1]: http://www.cs.technion.ac.il/~biham/Reports/Tiger/tiger/tiger.html
+//! [2]: https://crates.io/crates/digest
+
```

Comment:
It looks like you should be able to add `no_std` and our other attribute boilerplate without issue:

```suggestion
#![no_std]
```